front-dispatcher/src/components/dashboard/CardsContainer/DropDowns.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { apiStrings } from "../../../strings/apiStrings";
+import { searchBarStrings } from "../../../strings/searchBarStrings";
+import { Container } from "../../UI/Container/Container";
+import DropDown from "../../UI/DropDown/DropDown";
+
+interface props {
+  searchMainQuery: string;
+}
+const DropDowns = ({ searchMainQuery }: props) => {
+  const createDropDowns = () => {
+    if (searchMainQuery === searchBarStrings.searchDropDownOptions[0]) {
+      return apiStrings.TopHeadlinesOptions.map((cat) => {
+        if (cat === apiStrings.TopHeadlinesOptions[2]) {
+          return <DropDown options={[]} placeHolder={cat}></DropDown>;
+        }
+        //sources catagory case
+        if (cat && apiStrings[cat]) {
```

Comment:
I did not quite understand what was going on here

Reply:
My dropDown component will need to be completely different if the catagory is Dates
and also I didnt quite undertand how it should behave for the Catagory Sources(what it should show inside the dropdown as options)